app/views/wiki/edit_parent_page.html.erb

```diff
@@ -36,10 +36,10 @@ See doc/COPYRIGHT.rdoc for more details.
   <div class="box">
     <p>
       <%= f.select :parent_id,
-            "<option value=''></option>".html_safe + wiki_page_options_for_select(@parent_pages,
+            ("<option value=''>"+ I18n.t('js.placeholders.default') +"</option>").html_safe + wiki_page_options_for_select(@parent_pages,
```

Comment:
That line was already unnecessary before your change. Could you change that to:

```
[[I18n.t('placeholders.default'), '']] + wiki_page_options_for_select...
```

That will extend the available options with the empty option without explicitly rendering it.

Please also create the placeholder value `-`  in the Rails `en.yml` as well instead of using the js version. That keeps those two definitions separated